backend/seed_data/modules.yaml

```
@@ -187,3 +187,54 @@ modules:
     order: 2
     module_type: BONUS_SOLUTION
 
+# UNIT 5: Graphs
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)